editor/src/clj/editor/ns_batch_builder.clj

```diff
@@ -28,8 +29,9 @@
           (for [dep deps] [sym dep])))
 
 (defn- make-namespace-deps-graph
-  [srcdir]
-  (let [namespaces (ns-find/find-ns-decls [(io/file srcdir)])
+  []
+  (let [namespaces (ns-find/find-ns-decls
+                     (mapv io/file (.split (System/getProperty "java.class.path") File/pathSeparator)))
```

Comment:
Now we build the Clojure namespace dependency list for the whole classpath. This means Clojure dependencies are now split into batches too, instead of being loaded in parallel. This is necessary; otherwise, sometimes we'll get errors when loading e.g. `clojure.data.json`.